pkg/curatedpackages/curatedpackages.go

```diff
@@ -70,9 +74,22 @@ func PrintLicense() {
 	//the AWS Service Terms are extended to provide customers access to these features free of charge.
 	//These features will be subject to a service charge and fee structure at ”General Availability“ of the features.
 	//----------------------------------------------------------------------------------------------------------------
-	fmt.Println(strings.Repeat("-", width))
+	fmt.Println(strings.Repeat(constants.UserMsgSeparator, width))
 	fmt.Println(license)
-	fmt.Println(strings.Repeat("-", width))
+	fmt.Println(strings.Repeat(constants.UserMsgSeparator, width))
+}
+
+func PrintCertManagerDoesNotExistMsg() {
+	// Currently, use the width of the longest line to repeat the dashes
+	// Sample Output
+	//----------------------------------------------------------------------------------------------------------------
+	//Curated packages cannot be installed as cert-manager is not present in the cluster. This is most likely caused
+	//by an action to install Curated packages at a workload cluster. Refer to
+	//https://anywhere.eks.amazonaws.com/docs/tasks/packages/ for how to resolve this issue.
+	//----------------------------------------------------------------------------------------------------------------
+	fmt.Println(strings.Repeat(constants.UserMsgSeparator, width))
```

Comment:
is it worth to store the separator in a const/variable? It seems like we know what this need to be at compile time